utils/data-migration/tfm/migrate-gef-deals.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * DATA MIGRATION
+ * **************
+ * Purpose of this script is to migrate GEF deals to TFM.
+ * Since GEF deals are not subjected to execution on Workflow.
+ */
+const {
+  getCollection,
+  disconnect,
+} = require('../helpers/database');
+const {
+  getDDMMYYYY,
+  epochInSeconds,
+} = require('../helpers/date');
+const {
+  getDealTfmStage,
+} = require('../helpers/deal');
+const { ObjectId } = require('mongodb');
+const CONSTANTS = require('../constant');
+
+const version = '0.0.1';
+
+// ******************** DEALS *************************
+
+/**
+ * Return all the portal deals, without (default) or with filter specified.
```

Comment:
something to note on this, we don't need all deals from portal. only the ones that have been migrated